src/gatsby/src/components/park/parkHeader.js

```diff
@@ -107,36 +107,36 @@ export default function ParkHeader({
                 hidePunctuation={true}
               />
             </div>
-            <div className="park-header-child">
-              <FontAwesomeIcon icon={faCalendar} />
-              <div>
-                {parkDates ? (
+            {parkDates && (
+              <div className="park-header-child">
+                <FontAwesomeIcon icon={faCalendar} />
+                <div>
                   <p>
                     The {parkType} {parkOperation?.hasParkGate !== false && "gate"} is open {parkDates}.
                     {(parkOperation?.gateOpenTime && parkOperation?.gateCloseTime) && (
                       ` Gates are open from ${formattedTime(parkOperation.gateOpenTime)}
                        – ${formattedTime(parkOperation.gateCloseTime)}.`
                     )}
                   </p>
-                ) : (
-                  <p>Operating dates are unavailable.</p>
-                )}
-                {(campings.length > 0 || facilities.length > 0) && (
-                  <p>
-                    {campings.length > 0 &&
-                      <a href="#park-camping-details-container">Check campgrounds</a>
-                    }
-                    {facilities.length > 0 &&
-                      <>
-                        {campings.length > 0 ? " and " : ""}
-                        <a href="#park-facility-container">{campings.length > 0 ? "facilities" : "Check facilities"}</a>
-                      </>
-                    }
-                    {" "}for additional dates.
-                  </p>
-                )}
+                  {(campings.length > 0 || facilities.length > 0) && (
+                    <p>
+                      {campings.length > 0 &&
+                        <>
+                          Check <a href="#park-camping-details-container">campgrounds</a>
+                        </>
+                      }
+                      {facilities.length > 0 &&
+                        <>
+                          {campings.length > 0 ? " and " : "Check "}
+                          <a href="#park-facility-container">facilities</a>
```

Comment:
Should this anchor be renamed?

Reply:
Yes, it'll be fixed in another ticket: https://github.com/bcgov/bcparks.ca/pull/1384